pkg/server/container_update_resources.go

```diff
@@ -90,34 +93,22 @@ func (c *criContainerdService) updateContainerResources(ctx context.Context,
 				glog.Errorf("Failed to marshal spec %+v for container %q: %v", oldSpec, id, err)
 				return
 			}
-			info.Spec = any
-			if _, err := c.client.ContainerService().Update(ctx, info, "spec"); err != nil {
+			if err := cntr.Container.Update(ctx, func(ctx gocontext.Context, client *containerd.Client, c *containers.Container) error {
+				c.Spec = any
```

Comment:
Nit..  from the marshal to the call to center.Container.Update the code is duplicative with the only exception being which spec is being marshaled.. So, could be made into a update helper passing in the spec and returning one of the marshal error or the possible .Update error if Crosby's .Update is capable of generating an error outside the update function. But that could be done later...

Reply:
Will do.

Reply:
Done.